### ISSUE TITLE
Add method to delete expectation suite and expectation types

### ISSUE THREAD

**Issue by StevenMMortimer** [fivetran/great_expectations#658]
As of 0.7.7 there doesn't appear to be a method that will delete an expectation_suite or specific expectation_types in a suite. There are other CRUD methods like `list_expectation_suites()`, `get_expectation_suite()`, and `save_expectation_suite()` so it seems a little odd not to have a delete method. I understand that you can just delete the file in the folder structure, but if there are other methods to handle the creation then it seems natural to have a method to delete as well. 

In short, I am implying two feature requests with the first probably being much easier to implement: 

1. `delete_expectation_suite()` - removes the entire suite by name
2. `delete_expectation_types()` - removes just the expectations matching the supplied list of types

**Comment by jcampbell** [comment]
Thank you @StevenMMortimer : this is another one that just makes sense.
